src/main/java/com/scottlogic/librarygradproject/Services/BorrowService.java

```diff
@@ -33,6 +37,10 @@ public void borrow(long bookId, OAuth2Authentication authentication) {
         borrowRepository.save(loan);
     }
 
+    public void save(Borrow borrow) {
```

Comment:
is this not the same as the method above without the validation and authentication?

Reply:
its different, the one above uses bookID and authentication token to create a new borrow and save it. This one simply saves an existing Borrow into the Repo with no validation. At this stage I added it for testing but I imagine it might be useful for other things internally on backend (not exposed to frontend through an endpoint).

Reply:
If it's just for testing and serves no other purpose I think it should be removed, I don't see how we'd ever create a borrow object manually and add it without validation/authorisation?

Reply:
sure I can remove it, will just make the tests look more complicated